### PR TITLE
[MM-34183] - Getting Started Step 3 (Invite members to team) for new users on paid Workspace above 10 users are unable to invite members via email

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### Summary
Over limit errors shouldn't show in a paid workspace

#### Ticket Link
[MM-34183](https://mattermost.atlassian.net/browse/MM-34183?atlOrigin=eyJpIjoiZTRjYjhhYjAxYWVkNDkzM2IyOTZiOGVlODAyNjViNGEiLCJwIjoiaiJ9)

### CODE REVIEW

File: components/next_steps_view/steps/invite_members_step/invite_members_step.tsx [hunk 1]
```diff
@@ -127,12 +140,14 @@ class InviteMembersStep extends React.PureComponent<Props, State> {
         if (value.indexOf(' ') !== -1 || value.indexOf(',') !== -1) {
             const emails = value.split(/[\s,]+/).filter((email) => email.length).map((email) => ({label: email, value: email, error: !isEmail(email)}));
             const newEmails = [...this.state.emails, ...emails];
-            const {subscriptionStats, cloudUserLimit} = this.props;
+            const {cloudUserLimit} = this.props;
+
+            const showError = this.shouldShowLimitError(newEmails.length);
```
Comment (pablovelezvidal) [inline components/next_steps_view/steps/invite_members_step/invite_members_step.tsx:145]: just wondering if wouldn't it be clearer to call this constant showLimitError? I think that showError is too generic

Comment (AGMETEOR) [reply components/next_steps_view/steps/invite_members_step/invite_members_step.tsx]: Hey @pablovelezvidal thanks for the review. I'll take the suggestion. I have changed it to ```showLimitError```

**Review by marianunez — APPROVED** [review]
LGTM, nice tests! Minor comments below

**Review by stevemudie — APPROVED** [review]
Will test post merge

**Pull request merged**